src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: "Kubernetes integration: what's changed in v3"
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Changelog
+metaDescription: Changes introduced in Kubernetes Integration version 3
+---
+
+<Callout title="v3 BETA">
+  Kubernetes integration v3 is currently in a last stage beta, and we expect to make it generally available during Winter 2022. We encourage you to give it a try and let us know what you think!
```

Comment:
Winter 2022? I would not specify any date

Reply:
If we don't, users might get scared and think it's not worth jumping into the beta :(
Maybe we can hear @mangulonr's opinion here

Reply:
Changed to Spring 2022 as per Mikel's suggestion